graphql/schemabuilder/pagination.go

```diff
@@ -149,6 +149,14 @@ type PaginationInfo struct {
 	Pages          []string
 }
 
+// PostProcessOptions can be returned in a PaginateFieldFunc as an argument that succeeds
+// PaginationInfo. When a PaginationInfo is set, it must be succeeded by a PostProcessOptions.
```

Comment:
nit: I found this description a little confusing ("succeeds" has multiple meanings).   Maybe something like: "This struct should be returned as the second result from a manually-paginated field func"?